ex1/buttonActions.js

```diff
@@ -1,38 +1,42 @@
 
-var indexOfColor = 1;
+const INITIAL_COLOR = 0;
+var indexOfColor = INITIAL_COLOR;
 var colorArray = ['blue', 'red','orange', 'yellow'];
+var color = window.localStorage.getItem("color");
 //A function that changes the color of button on every click
 //If we got to the last color then an image will appear.
 function changeColor() {
-    setTheColor(indexOfColor);
-    var noaImage = document.getElementById("noaIMG");
+    setTheColor();
+    var noaImageElement = document.getElementById("noa-img");
```

Comment:
Everything that's got to do with this element please extract to a separate function that will handle its logic